scripts/run.sh

```diff
@@ -17,5 +18,24 @@ if [ ! -d ${APACHEDS_INSTANCE_DIRECTORY} ]; then
     chown -v -R ${APACHEDS_USER}:${APACHEDS_GROUP} ${APACHEDS_INSTANCE_DIRECTORY}
 fi
 
-# Execute the server in console mode and not as a daemon.
-/opt/apacheds-${APACHEDS_VERSION}/bin/apacheds console ${APACHEDS_INSTANCE}
+cleanup(){
+    if [ -e "${PIDFILE}" ];
+    then
+        echo "Cleaning up ${PIDFILE}"
+        rm "${PIDFILE}"
+    fi
+}
+
+trap cleanup EXIT
+cleanup
+
+/opt/apacheds-${APACHEDS_VERSION}/bin/apacheds start ${APACHEDS_INSTANCE}
+sleep 2  # Wait on new pid
+
+shutdown(){
+    echo "Shutting down..."
+    /opt/apacheds-${APACHEDS_VERSION}/bin/apacheds stop ${APACHEDS_INSTANCE}
+}
+
+trap shutdown INT TERM
+tail --pid=$(cat $PIDFILE) -f /dev/null
```

Comment:
There's also
- `/var/lib/apacheds/default/log/apacheds.log`
- `/var/lib/apacheds/default/log/wrapper.log`
Is one of these more useful?

Reply:
Neither is good, but that's likely a log4j.properties issue. I'll use the more specific apacheds.log.